features/metadata.feature

```diff
@@ -38,22 +38,42 @@ Feature: Metadata
 
     Scenario Outline: Adding some data to metadata
         Given an existing pipeline with the workflow:
-            | job  | triggers |
-            | main | BAR      |
-            | BAR  | BAZ      |
+            | job   | triggers  |
+            | main  | BAR       |
+            | BAR   | BAZ       |
+            | BAZ   | BOOZ      |
+            | BAM   |           |
+        And the "BOOZ" job is disabled
         When the "main" job is started
-        Then add the { "foo": <foobar> } to metadata in the "main" build container
         And the build succeeded
+        Then add the { "foo": <foobar> } to metadata in the "main" build container
         And the "BAR" job is started
         Then in the build, the { "foo": <foobar> } is available from metadata
-        And add the { "bar": <barbaz> } to metadata in the "BAR" build container
         And the build succeeded
+        And add the { "bar": <barbaz> } to metadata in the "BAR" build container
```

Comment:
same comment, should this be moved above `And the build succeeded`

Reply:
This is the right order, the buildMeta is needed in order to check the metadata values. 
https://github.com/screwdriver-cd/screwdriver/blob/65f699b61cbcaf6e1b35dac44fb5a01e10ca0313/features/step_definitions/metadata.js#L96

Reply:
Syntactically it doesn’t make sense that you add the metadata after the build is already done. Should be more like “and the metadata is added”

Reply:
I will make sure that this is fixed through the next PR.